src/functionalTest/resources/application.yml

```diff
@@ -69,6 +69,10 @@ azure:
       container-url: ${CVP_STORAGE_URL:https://emhrsapiaat.blob.core.windows.net/cvpsimulator/}
       connection-string: ${CVP_STORAGE_CONNECTION_STRING:DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://localhost:10000/devstoreaccount1}
       blob-container-reference: ${CVP_STORAGE_CONTAINER_NAME:cvptestcontainer}
+    vh:
+      container-url: ${CVP_STORAGE_URL:https://emhrsapiaat.blob.core.windows.net/cvpsimulator/}
+      connection-string: ${CVP_STORAGE_CONNECTION_STRING:DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://localhost:10000/devstoreaccount1}
+      blob-container-name: ${CVP_STORAGE_CONTAINER_NAME:cvptestcontainer}
```

Comment:
Should this be vhtestcontainer?

Reply:
for now using cvp , VH vars not created in jenkins and  tests env.